### DISCUSSION TITLE
Installation in Windows Localhost

### DISCUSSION THREAD

**Discussion by gokulmhegde** [requarks/wiki#3468] [Help / Questions]
I am trying to install to Wiki.js on windows 10 localhost. I am getting the following error

(node:11360) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'toString' of null
    at Object.init (C:\wiki\server\core\db.js:35:37)
    at Object.init (C:\wiki\server\core\kernel.js:13:35)
    at Object.<anonymous> (C:\wiki\server\index.js:35:13)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
(node:11360) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:11360) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

I have following : 
node version **v12.16.1** 
npm version **6.13.4**
MySQL **Ver 15.1 Distrib 10.4.11-MariaDB, for Win64 (AMD64)**, source revision 7c2c420b70b19cc02b5281127205e876f3919dad

Any help is appreciated

**Comment by Frankstar** [comment]
did you use [this documentation?](https://docs.requarks.io/install/windows)
wikijs 2.5.159 ?

Edit: can you try using a different node version? like a 14.x ?

**Comment by gokulmhegde** [comment]
> did you use [this documentation?](https://docs.requarks.io/install/windows)
> wikijs 2.5.159 ?
> 
> Edit: can you try using a different node version? like a 14.x ?

I used the same documentation which you mentioned.

I tried same not worked

**Comment by jaypeeperos** [comment]
experiencing same issue. Still trying to find fixes.

**Comment by jaypeeperos** [comment]
This solution is working for me: https://github.com/Requarks/wiki/issues/2302#issuecomment-676233715
